package.json

```diff
@@ -82,16 +83,18 @@
   },
   "scripts": {
     "once": "npm run bundle && npm run firefox",
-    "clean": "rm -rf data/content/*",
+    "setup": "npm-run-all setup:*",
+    "setup:clean": "rm -rf data/content/* && mkdir -p data/content",
+    "setup:externals": "cat node_modules/react/dist/react.min.js node_modules/react-dom/dist/react-dom.min.js node_modules/moment/min/moment.min.js > data/content/vendor.js",
```

Comment:
We *could*, but last time I tried, for some reason I could never get react down to as small a size as their own minified version is...